annotations/android-places/metrics/places_manager.read_query_count/README.md

```diff
@@ -1,3 +1,3 @@
-Read and write success rates for the places store can be found in the bottom half of [STMO#fenix-logins-and-places-performance-metrics (SSO)](https://sql.telemetry.mozilla.org/dashboard/fenix-logins-and-places-performance-metrics?p_org_mozilla_fenix=org_mozilla_firefox).
+Read and write success rates for the places store can be found in the bottom half of https://sql.telemetry.mozilla.org/dashboard/fenix-logins-and-places-performance-metrics?p_org_mozilla_fenix=org_mozilla_firefox.
```

Comment:
This output doesn't look like I'd expect. I think the regex has some extra escaping that needs to happen.

Reply:
I think I'm confused about intent here. I guess these are manual edits you've made, and the substitutions only happen to rendered output.

Reply:
Correct, I mass updated the files via a regex replace. This is definitely an issue with the regex, will fix